index.html

```diff
@@ -1868,12 +1868,11 @@ <h3>Identifier Restrictions</h3>
     <section>
       <h2>Verification Methods</h2>
       <p>
-A [=DID document=] can express [=verification methods=], as defined in
-<a data-cite="CID#verification-methods">Section 2.2: Verification Methods</a> of
-[[[CID]]] with the added restriction that the `id` value
-MUST conform to the [[[#did-url-syntax]]] and the
-`controller` value MUST conform to the [[[#did-syntax]]].
-See
+A [=DID document=] can express [=verification methods=], as defined in Section
+<a data-cite="CID#verification-methods">2.2: Verification Methods</a> of
+[[[CID]]] with the added restriction that the `id` value MUST conform to Section
+[[[#did-url-syntax]]] or Section [[[#relative-did-urls]]] and the `controller`
+value MUST conform to the Section [[[#did-syntax]]]. See
 <a data-cite="CID#verification-methods">Section 2.2: Verification Methods</a>
```

Comment:
Wondering if we still need this end bit given the above. Especially

> A [=DID document=] can express [=verification methods=], as defined in Section
<a data-cite="CID#verification-methods">2.2: Verification Methods</a> of
[[[CID]]]